shared/js/loadApp.js

```diff
@@ -29,7 +29,9 @@ module.exports = function (callback) {
     }
   };
 
-  if (!appOptions.channel) {
+  var isViewingSolution = (dashboard.clientState.queryParams('solution') === 'true');
```

Comment:
the rails code tests for the presence of params['solution'] not that it === 'true' so this is slightly different from the condition there...